Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM openshift/origin-base
+
+ENV APP_HOME=/opt/app/src \
+    GOPATH=/go \
+    PATH="${PATH}:${GOPATH}/bin:/usr/local/bin" \
+    INSTALL_PKGS="golang make git"
+
+RUN yum install -y ${INSTALL_PKGS} --setopt=tsflags=nodocs
+
+WORKDIR /tmp/cluster-logging-operator
+COPY vendor/ /tmp/cluster-logging-operator/vendor
```

Comment:
do we need to copy these? won't they be in our current directory when we `RUN make`?

Reply:
The build happens in context of the Dockerfile so all the code needs to be copied there.  We are able to `make` outside of the context of Docker and `make image` inside the context of Docker.  The instructions in the Dockerfile will remove all build sources and build tools.